Tic-Tac-Toe.py

```diff
@@ -0,0 +1,171 @@
+#Tic Tac Toe
+
+#create a 3x3 list of blank spaces
+def new_board():
+    board = [[" ", " ", " "], [" ", " ", " "], [" ", " ", " "]]
+    return board
+
+#print the current board to the user
+def print_board(board):
+    print("", board[0][0], "|", board[0][1], "|", board[0][2])
+    print("---+---+---")
+    print("", board[1][0], "|", board[1][1], "|", board[1][2])
+    print("---+---+---")
+    print("", board[2][0], "|", board[2][1], "|", board[2][2])
+
+#take the user input and return an X or O depending on who's turn it is
+def next_turn(turn):
+    if (turn % 2) == 0:
+        print("X's turn!")
+    elif (turn % 2) == 1:
+        print("O's turn!")
+    move = input("Enter your move: ")
+    print()
+    if turn%2 == 0:
+        return ("X", move)
+    else:
+        return ("O", move)
+
+#make sure the user input a valid entry and the space is not already taken
+def check_move(move, board):
+    if move[1] == 'a1':
+        if board[0][0] == " ":
+            return True
+        else:
+            print("That spot is already taken!")
+            print()
+            return False
+    elif move[1] == 'a2':
+        if board[0][1] == " ":
+            return True
+        else:
+            print("That spot is already taken!")
+            print()
+            return False
+    elif move[1] == 'a3':
+        if board[0][2] == " ":
+            return True
+        else:
+            print("That spot is already taken!")
+            print()
+            return False
+    elif move[1] == 'b1':
+        if board[1][0] == " ":
+            return True
+        else:
+            print("That spot is already taken!")
+            print()
+            return False
+    elif move[1] == 'b2':
+        if board[1][1] == " ":
+            return True
+        else:
+            print("That spot is already taken!")
+            print()
+            return False
+    elif move[1] == 'b3':
+        if board[1][2] == " ":
+            return True
+        else:
+            print("That spot is already taken!")
+            print()
+            return False
+    elif move[1] == 'c1':
+        if board[2][0] == " ":
+            return True
+        else:
+            print("That spot is already taken!")
+            print()
+            return False
+    elif move[1] == 'c2':
+        if board[2][1] == " ":
+            return True
+        else:
+            print("That spot is already taken!")
+            print()
+            return False
+    elif move[1] == 'c3':
+        if board[2][2] == " ":
+            return True
+        else:
+            print("That spot is already taken!")
+            print()
+            return False
+    else:
+        print("That is not a valid entry!")
+        print()
+        return False
+
+#takes the current board and the user's move as parameters and updates the board with the new move
+def update_board(board, player_move):
+    if player_move[1] == 'a1':
```

Comment:
Same comment here. If I find myself writing a lot of if/else blocks, there is often a more concise way.